frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -13,8 +14,20 @@ export type KnativeResourceOverviewPageProps = {
 };
 
 const KnativeOverview: React.FC<KnativeResourceOverviewPageProps> = ({ item }) => {
+  const { obj, current } = item;
   return (
     <div className="overview__sidebar-pane-body resource-overview__body">
+      {obj.kind === RevisionModel.kind && (
+        <div className="resource-overview__pod-counts">
+          <PodRing
+            pods={current ? current.pods : []}
```

Comment:
is there a possibility of current is defined but `pods` getting undefined. Check would help

Reply:
@invincibleJai no, If `current` is defined pods are going to be an empty array if no pods are available